dallinger/command_line/docker_ssh.py

```diff
@@ -392,17 +439,25 @@ def deploy(
         "dallinger/Caddyfile",
     )
 
-    print("Launching http and postgresql servers.")
+    dotenv_values = get_dotenv_values(executor)
+    if dotenv_values:
+        if dotenv_values.get("DOZZLE_PASSWORD"):
```

Comment:
Better to simplify to this?

```py
dozzle_password = dotenv_values.get("DOZZLE_PASSWORD", secrets.token_urlsafe(8))
```

but that said, how come we use `token_urlsafe` instead of the dashboard password in this case?

Reply:
> We should use a randomized passowrd. It can be stored [in a file named](https://docs.docker.com/compose/environment-variables/env-file/) .env next to the docker compose file.

This comment lead me to think that we wanted a randomized password if a .env file existed but without a `DOZZLE_PASSWORD` defined.
I simplified the implementation to default to the dashboard password and this looks much cleaner now.

Reply:
Great!